docs/REPO-LAYOUT.md

```
@@ -0,0 +1,144 @@
+  dev/     run-dev-test.sh clean-profile.py compare_json.py
+```
+
+Reference-update surface (grep each moved basename before+after):
+`Makefile`, `.github/workflows/*.yaml`, `local-ci*.sh`, `README.md`.
+Verify with `scripts/lib/test/run.sh` + a `grep -rn '<oldpath>'` gate
+returning empty. Contained; one PR.
+
+### Tier C — per-app `example/<app>/` + fold root app dirs (high-touch, high-value)
+
+Target:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)